Add the text article Infinite Loop to my article

SakshamGupta1995 opened the pull request:
The article  "##Infinite Loop by opposite iteration##" gives an example to new java programmer as to how a program can go into infinite loop if we increment the value of i instead of decrementing it and vice-versa. It is a basic but an important example for new coders.

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

SakshamGupta1995 committed:
Add the text article Infinite Loop to my article

The article  "##Infinite Loop by opposite iteration##" gives an example to new java programmer as to how a program can go into infinite loop if we increment the value of i instead of decrementing it and vice-versa. It is a basic but an important example for new coders.

Manish-Giri reviewed (changes requested):
1.  To add more code samples demonstrating infinite loops, use a statement to introduce the same with the `h4` tag (follow the recommended guide formatting), like:
 
     > #### Some more examples of infinite loops are shown below.

2. Remove the `/*My Changes` line

3. Use subheadings like `h4` to introduce points like `##Infinite Loop by opposite iteration##` and remove the hashes at the end.

4. Please don't add unnecessary new lines and remove the ones you've added.

SakshamGupta1995 committed:
Update index.md

SakshamGupta1995 committed:
Add Infinite Loop by update

This article will give new programmers the example of infinite loop when we increment the value instead of decrementing and vice-versa.

Manish-Giri committed:
Fixed several formatting issues

cmccormack commented:
Closing and reopening to reset CI

cmccormack commented:
Looks good to me, and congratulations on your first Pull Request!  🎉🎉🎉
We look forward to future contributions!
